src/main/java/org/apache/sling/resourceresolver/impl/ResourceResolverImpl.java

```diff
@@ -1046,37 +1051,57 @@ public String getParentResourceType(final String resourceType) {
     public boolean isResourceType(final Resource resource, final String resourceType) {
         boolean result = false;
         if ( resource != null && resourceType != null ) {
-             // Check if the resource is of the given type. This method first checks the
```

Comment:
This inline comment seems to have been lost in the refactoring.

Reply:
Added it as method description for `isResourceTypeInternal`